cv32/sim/uvmt_cv32/xrun.mk

```diff
@@ -21,29 +21,107 @@
 #
 ###############################################################################
 
-XRUN              = xrun
+XRUN              = $(CV_SIM_PREFIX) xrun
+SIMVISION         = $(CV_TOOL_PREFIX) simvision
+INDAGO            = $(CV_TOOL_PREFIX) indago
+IMC               = $(CV_SIM_PREFIX) imc
+
 XRUN_UVMHOME_ARG ?= CDNS-1.2-ML
 XRUN_COMP_FLAGS  ?= -64bit -disable_sem2009 -access +rwc -q -clean \
                    -top uvmt_cv32_tb -sv -uvm -uvmhome $(XRUN_UVMHOME_ARG) \
                    $(TIMESCALE) $(SV_CMP_FLAGS)
-XRUN_DIR         ?= xcelium.d
 XRUN_GUI         ?=
+XRUN_RESULTS     ?= $(PWD)/xrun_results
+XRUN_RISCVDV_RESULTS ?= $(XRUN_RESULTS)/riscv-dv
+XRUN_DIR         ?= $(XRUN_RESULTS)/xcelium.d
 XRUN_SINGLE_STEP ?=
-#XRUN_VELABCOVERAGE = -covdut uvmt_cv32_tb -coverage function
-#XRUN_VRUNCOVERAGE  = -covoverwrite -covtest uvmt_cv32_tb -covscope uvmt_cv32_tb
-# VCOVERAGE     = imc -load cov_work/$(TESTBENCH)/$(TESTBENCH) -execcmd "report -summary"
+XRUN_ELAB_COV     = -covdut uvmt_cv32_tb -coverage b:e:f:t:u
+XRUN_RUN_COV      = -covoverwrite -covscope uvmt_cv32_tb
+NUM_TEST         ?= 2
+
+################################################################################
```

Comment:
I get what you are trying to do here, and have no objections to the concepts.  A couple of questions:
1. What makes `ifeq ($(call IS_YES, ($MYVAR)),YES)` better? (Both are terrible, but that's "make" for you.)
2. Do we know this will work across multiple Cadence installations?

Reply:
1. I'm certainly not proud of that, but at least it kept the polarity of the statement consisten (i.e. that are no negations in the clause)
Taking the statement above, the other alternative I came up with was something like:
ifneq ($filter $(YES_VALS),$(MYVAR),)
# This is the positive clause!

Mentally I didn't like seeing a ifneq on a positive YES test. 

2. Fair question regarding Cadence.  I did update a couple of internal -execcmds to IMC because of deprecated commands.  In general what has worked reasonably well for me is to just put a stake in the ground (using a latest release or only one major release behind) and not be so aggressive about updating with latest and greatest.  Again this is only trying to support the absolute basics and I would expect certain power users to replace these in their own wrappers (should probably make these ?= now that I am thinking about this).

Reply:
1. Yes, we all _love_ Make.  :-)

2. I like the "stake in the ground" approach, so let's proceed.  In an abundance of caution I will ask another Contributor who uses Xcelium (@alfredoh1234) to review before merging.